CoCSharp/Network/Messages/ChatMessageServerMessage.cs

```diff
@@ -95,16 +95,13 @@ public override void WriteMessage(MessageWriter writer)
             writer.Write(League);
             writer.Write(UserID);
             writer.Write(CurrentUserID);
-
+            writer.Write(Clan == null ? false : true);
```

Comment:
Yep?Changed in my local...Will push within 5 minute as currently updating network driver on oc